docs/console/company-configuration/extensions.md

```diff
@@ -68,35 +71,112 @@ On this page, you can:
 
 ## Extension Detail
 
-The extension detail page allows you to view the main information that characterizes the registered extension, which can also be modified through the corresponding *Edit* buttons.
+The extension detail page allows you to view the main information that characterizes the registered extension, which can also be modified through the corresponding _Edit_ buttons.
 
 ![extension detail](./img/extensions/extension-detail.png)
 
-### Extension Info
+It includes several cards to provide information about the extension, such as:
+
+- **Extension Info**: contains the identifying information for the extension, such as name and entry URL
+- **Extension Location**: contains the information related to the positioning of the extension menu item within the Console;
+- **Extension Visibility**: contains the information related to the visibility of the extension menu item within the Console;
 
-The *Extension Info* card contains the identifying information for the extension, where only the name and entry URL can be edited.
+In each card, you can also find the _Edit_ button, which allows you to modify the information contained in the card.
 
-![edit extension info](./img/extensions/edit-extension-info.png)
+:::info
+Just as during creation, Visibility is tied to the Destination Area set for the extension.
+For example, if the Destination Area is set to _Company Overview_, the only available visibility option will be _Whole Company_.
+:::
 
 ### Extension Configuration
 
-If the current extension is of type Composer page, the badge will display the *Edit page with MFE Composer* button which brings the user to a page where it can edit the extension's `micro-lc` configuration.
+If the current extension is of type Composer page, the badge will display the _Edit page with Composer_ button which brings the user to a page where it can edit the extension's `micro-lc` configuration.
 
 ![edit extension configuration](./img/extensions/edit-extension-configuration.png)
 
-### Extension Location
+## Extension Communication with APIs
 
-The *Extension Location* card contains the information related to the positioning of the extension menu item within the Console; these details are fully editable.
+:::info
+This section is only available for extensions of type _Composer_.
+:::
 
-![edit extension destination](./img/extensions/edit-extension-destination.png)
+<!-- TODO: Do we have here a link with a comprehensible list of these APIs? -->
+It is also possible to configure the extension to communicate with other external APIs, as well with the Mia-Platform supported APIs.
```

Comment:
I wish to add a link to a comprehensible list of APIs here. I don't know if there's one yet, any suggestion on how better rephrase this to emphasize the concept of internal APIs would be greatly appreciated.